src/auth_api/orchestrator.py

```diff
@@ -171,8 +180,9 @@ def _log_in_user_and_create_cookie(
         self
     ) -> Cookie:
         """
-        Register user login after completed registration and create 
-        httponly cookie
+        Register user login and creates cookie.
```

Comment:
```suggestion
        Register user login and creates cookie.

```
Add new line between summary and extended summary.